packages/url_launcher/url_launcher/README.md

```diff
@@ -53,6 +53,29 @@ Common schemes supported by both iOS and Android:
 
 More details can be found here for [iOS](https://developer.apple.com/library/content/featuredarticles/iPhoneURLScheme_Reference/Introduction/Introduction.html) and [Android](https://developer.android.com/guide/components/intents-common.html)
 
+### Encoding URLs
+
+URLs should be safely encoded, espeically when including spaces or other special characters. We can do this using dart's `Uri` helper methods included in `dart:core`:
+```dart
+import 'dart:core';
+import 'package:url_launcher/url_launcher.dart';
+
+final Uri _emailLaunchUri = Uri(
+  scheme: 'mailto',
+  userInfo: 'support',
+  host: 'flutter.dev',
```

Comment:
The use of `userInfo`/`host` here is not correct for `mailto`, per the comment linked earlier in this discussion. Per your own testing, this would fail.